tests/Tests/100__deploy/100__installation/109__operator.robot

```diff
@@ -47,6 +47,60 @@ Verify That The Operator Pod Does Not Get Stuck After Upgrade
             Log Error And Fail Pods When Pods Were Terminated    ${operator_pod_info}    Opertator Pod Stuck
         END
     END
+Verify Clean Up ODS Deployer Post-Migration
+    [Tags]    Tier1
+    ...       ODS-1767
+    ${version_check} =    Is RHODS Version Greater Or Equal Than    1.17.0
+    IF    ${version_check} == False
+        Log    Skipping test case as RHODS version is less than 1.17.0
+        Skip
+    END
+    ${cro_pod}=    Run Keyword And Return Status
+    ...    Oc Get    kind=Pod    api_version=v1    label_selector=name=cloud-resource-operator
+    IF    ${cro_pod} == True
+        Fail    CRO pod found after migration
+    END
```

Comment:
Failing here means the rest of the TC isn't executed.
Would it be more useful to keep going and collect all potential failures other than the presence of the CRO?

Reply:
Maybe `Run Keyword And Continue On Failure     Verify CRO Is Not Deployed` could have worked better. Also, we have the keyword  `Check If POD Exists` in Robots.robot that could have been used.

I'm going to merge this PR anyway as it is urgent